client/src/components/Question.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+// Input to search database for specific presentation
```

Comment:
This comment needs to be taken out and replaced with a description of this module